src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/VisitDetailsClient.kt

```diff
@@ -3,10 +3,12 @@ package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.client
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Component
 import reactor.core.publisher.Mono
-import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.hmpps.auth.UserDetails
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.hmpps.auth.UserDetailsDto
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.orchestration.VisitHistoryDetailsDto
 import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.VisitDto
 import java.time.Duration
-import java.util.Optional
+import java.time.LocalDateTime
+import java.util.*
```

Comment:
not sure how the lint checks didn't pick this.

Reply:
You mean the .* I don't actually think it should be checking for such things

Reply:
ok.. I thought it fails lint check if we do a *. But yes can mark it resolved for now. Can change if it gets picked at a alter stage.